Examples/swiftui-toot/SwiftUI-Toot.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,16 +6,16 @@
       "location" : "https://github.com/divadretlaw/EmojiText",
       "state" : {
         "branch" : "main",
-        "revision" : "510413aeccc50891cf74d6cb6b273125190cdd3c"
+        "revision" : "263935972835482fe655d7ce44e101cc3fb463bf"
```

Comment:
Let's pin the dependency to the older commit `510413aeccc50891cf74d6cb6b273125190cdd3c` for now. I see EmojiText is making some internal changes that impact layout and caching and should be tested before upgrading.

Reply:
I also requested that they tag their versions without the "v" prefix so we can use them as a dependency instead https://github.com/divadretlaw/EmojiText/issues/1